lib/config.py

```diff
@@ -66,6 +66,9 @@ def parse_itemsdir(itemsdir, item_conf):
         if item_file.endswith(CONF_FILE) or item_file.endswith(YAML_FILE):
             try:
                 item_conf = parse(itemsdir + item_file, item_conf)
+            except UnicodeDecodeError as e:
+                logger.error("item file ({}) is not in UTF-8 format, please check character encoding at position {}! => ignoring config".format(item_file, e.start))
+                continue
```

Comment:
While in transition from config to yaml this seems to be a quick pointer for unexpierienced users to show that the problem is within their saving files with wrogn encoding. It will catch config and yaml as well.
The parse yaml code looks clean but the parse config is a mess and doesn't contain any error handling at all. So if we drop config soon, there is no need to improve and fix that code and I am pretty sure that @msinn will soon include some more error handling for yaml if needed.